jojo.py

```diff
@@ -543,6 +640,61 @@ def update_all_ticketcompass(world,tusk,kqeen,rain,boy,feat):
     boy.create_ticket_compass()
     feat.create_ticket_compass()
 
+def stand_list_json_rewrite_for_new_joinner(mcr):
+    '''
+    スタンドを持っていない新規参入者のためにstand_list.jsonを読み取り、空きがあれば書き込む処理
+
+    Parameter
+        mcr : MCRcon
+            Rconのサーバ情報
+
+    Return
+        bool
+    '''
+    #スタンドリストの格納場所
+    str_dir = 'json_list'
+    str_stand_file = 'stand_list.json'
+    
+    #「NEW」が付与されているアマスタのTag(プレイヤー名)を取得
+    resp = mcr.command(f'/data get entity @e[limit=1,type=minecraft:armor_stand, name=NEW] Tags')

```

Comment:
こっちもスラッシュを消してほしい。
あとname=NEWの前のスペースも無くしてほしい。
```suggestion
    resp = mcr.command(f'data get entity @e[limit=1,type=minecraft:armor_stand,name=NEW] Tags')

```

Reply:
拡張機能を使用するように修正しました